tests/test_chat_history.py

```
@@ -21,44 +21,45 @@ class TestGetChatHistory(unittest.TestCase):
+
         self.get_chat_history = get_chat_history
 
     def test_returns_same_instance_regardless_of_session_id(self):
         """get_chat_history should return the same global memory for any session_id."""
-        history_a = self.get_chat_history('session-1')
-        history_b = self.get_chat_history('session-2')
-        history_c = self.get_chat_history('different')
+        history_a = self.get_chat_history("session-1")
+        history_b = self.get_chat_history("session-2")
+        history_c = self.get_chat_history("different")
         self.assertIs(history_a, history_b)
```

Ruff (0.15.6)

[warning] Use a regular `assert` instead of unittest-style `assertIs`

Replace `assertIs(...)` with `assert ...`

(PT009)